### PR TITLE
Collapse ShootList Columns

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
This PR introduces a new component `CollapsibleItems` that can be used to collapse items where the space is limited

Screenshot 2024-07-18 at 16 49 56

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user
To enhance the overview and readability of the cluster list, particularly in environments constrained by space or containing an abundance of information, we have introduced a feature that allows items to be collapsed when they are displayed in the cluster list.
```

### CODE REVIEW

File: frontend/src/components/GStatusTag.vue [hunk 1]
```diff
@@ -225,7 +225,7 @@ export default {
     },
     color () {
       if (this.isUnknown || this.staleShoot) {
-        return 'unknown'
+        return 'grey'
```
Comment (holgerkoser) [inline frontend/src/components/GStatusTag.vue:228]: Should be unknown

File: frontend/src/components/ShootWorkers/GWorkerGroup.vue [hunk 2]
```diff
@@ -10,28 +10,25 @@ SPDX-License-Identifier: Apache-2.0
...
+        <g-vendor-icon
+          :icon="machineImage.icon"
+          :size="20"
+        />
+        <span class="pl-1">{{ workerGroup.name }}</span>
+        <v-tooltip
+          location="top"
+          :disabled="!machineImage.isDeprecated"
+          text="Machine image version is deprecated"
+        />
```
Comment (petersutter) [inline frontend/src/components/ShootWorkers/GWorkerGroup.vue:30]: tooltip does not work as activator is missing

File: frontend/src/views/GMembers.vue [hunk 3]
```diff
@@ -197,6 +197,9 @@ SPDX-License-Identifier: Apache-2.0
         :custom-key-sort="disableCustomKeySort(visibleServiceAccountTableHeaders)"
         class="g-table"
       >
+        <template #headers="headers">
+          <g-data-table-header v-bind="headers" />
+        </template>
```
Comment (petersutter) [inline frontend/src/views/GMembers.vue:202]: rm

File: frontend/src/views/GSecrets.vue [hunk 4]
```diff
@@ -222,6 +222,9 @@ SPDX-License-Identifier: Apache-2.0
         density="compact"
         class="g-table"
       >
+        <template #headers="headers">
+          <g-data-table-header v-bind="headers" />
+        </template>
```
Comment (petersutter) [inline frontend/src/views/GSecrets.vue:227]: rm

**Review by petersutter — APPROVED** [review]
/lgtm

**Review by holgerkoser — APPROVED** [review]
/lgtm only a minor renaming comment

File: frontend/src/components/GCodeBlock.vue [hunk 5]
```diff
@@ -66,6 +66,11 @@ export default {
       type: [Number, String],
       default: '450px',
     },
+    width: {
```
[the remark this answers is not in the document]
Comment (grolu) [reply frontend/src/components/GCodeBlock.vue:69]: done

**Pull request merged**